tests/unit/test_matcher.c

```diff
@@ -337,23 +339,22 @@ Test(matcher, test_replace_works_on_matches_input_with_store_matches_too)
   LogMessage *msg = log_msg_new_empty();
   log_msg_set_value_by_name(msg, "1", test_message, -1);
 
-  GlobalConfig *cfg = cfg_new_snippet();
-  replace_template = log_template_new(cfg, NULL);
+  replace_template = log_template_new(configuration, NULL);
   cr_assert(log_template_compile(replace_template, replace_pattern, NULL));
 
   NVHandle input_handle = log_msg_get_value_handle("1");
   const gchar *input = log_msg_get_value(msg, input_handle, &value_len);
 
+  NVTable *payload = nv_table_ref(msg->payload);
   const gchar *result = log_matcher_replace(m, msg, input_handle, input, value_len,
```

Comment:
Thanks for the fixup! I'm sorry, but I forgot to free the result string from `log_matcher_replace`...

Reply:
fixed.